Wiki/FAQ.md

```diff
@@ -0,0 +1,652 @@
+---
+title: FAQ
+layout: wiki
+---
+
+# Most frequent asked questions
+
+## I am crunching for the pool but the wallet still shows INVESTOR and 0 magnitude. Is that ok?
+
+That is perfectly normal, when pool-mining you do work for the pool's
+account and as a reward get transactions from the pool for the work you
+did. Because of this you do not work for your own CPID and have no
+magnitude.
+
+## I am crunching for the pool but even though I work on projects this is not shown on the profile page of the pool. What can I do?
+
+Make sure you remove all existing projects from the BOINC manager before
+syncing to the pool. Then you can add new projects from the page of the
+pool.
+
+## I have connections but the number of blocks does not increase. What can I do?
+
+If you have connections you should be fine and the wallet will sort
+itself out. If after a few hours your number of blocks still doesn't increase check to make sure you're using the latest version of the wallet.
+
+## I had a magnitude for some time but it dropped to 0. What happened?
+
+The reason for this is most likely that you have to resend your beacon,
+which should be done every 6 months. For this either unlock your wallet
+so that it sends automatically or type `advertisebeacon` in the
+console.
+
+# Gridcoin & Science
+
+## Why is Gridcoin (GRC) special?
+
+Gridcoin allows  Nearly all computing power (up to
+98% efficiency) can be directed at beneficial scientific and medical

```

Comment:
Is this 98 % still correct? I think it could be even more.

Reply:
I would remove the % comment as baseless conjecture